model/src/main/proto/platform_parameter.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// It will correspond to a single Platform Parameter Value.
+message PlatformParameter {
+    // Value of a parameter can be of any one of the following type.
+    oneof value {
+        // Boolean type parameter value.
+        bool boolean = 2;
+        // Integer type parameter value.
+        int32 integer = 3;
+        // String type parameter value.
+        string string = 4;
+    }
+}
+
+// Storage format of platform parameters while the app is running.
+message PlatformParameterMap{
+    map<string, PlatformParameter> parameter = 1;
+}
+
+// Pair of Platform Parameter Value and Name.
+message PlatformParameterWithName{
+    string name = 1;
+    PlatformParameter value = 2;
+}
+
+// Final storage format of parameters in the disk. This is made in the form of list so as to closely
+// resemble the JSON response in cache.
```

Comment:
Nit: ``Format of platform parameters stored on disk.`` Is probably sufficient. For the second part, do we specifically benefit from matching the JSON response? It seems unimportant here (though using a 'repeated' still seems fine regardless, but I don't think we care about retaining any particular order--do we?).

Reply:
Done.
No we don't need that specifically I just mentioned it because I think by following this structure we are keeping ourselves open to consume Parameters in any Data structure we want (currently a Map). Also Yes, this format will retain the order of parameters from the JSON.